### ISSUE TITLE
HackyHour[7] on 2016-06-02

### ISSUE THREAD

**Issue by iimog** [HackyHour/Wuerzburg#18]
discussion is open :smiley:

**Comment by iimog** [comment]
The weekday of future HackyHours is currently up for discussion. Please vote at #19

**Comment by iimog** [comment]
Thursday will be our new weekday for future HackyHours. As the last Thursday of May is holiday we will meet on June, 2nd.

**Comment by iimog** [comment]
Any ideas for the next HackyHour. Of course we will discuss our solutions to the "Skynet: The Chasm" challenge on [codingame.com](https://www.codingame.com/servlet/urlinvite?u=814777) But that will not take too long. If you want to solve it or play around with some other challenges sign up at: [codingame.com](https://www.codingame.com/servlet/urlinvite?u=814777)

**Comment by greatfireball** [comment]
Good idea... And this topic can fill more than one HackyHour @iimog :)

I will not attend the next HackyHour... Maybe, I will prepare a solution for _"Skynet: The Chasm"_ and you can try to understand it :)

Anyway... If you want to join you can also use [my invite link](https://www.codingame.com/servlet/urlinvite?u=1424800)

**Comment by EzioAuditoredaFirenze** [comment]
@iimog @greatfireball  I managed to solve the _"Skynet: The Chasm"_ challenge. I solved it using logical reasoning rather than the mathematical equation approach that we tried during the last HackyHour. I guess that the use of equations would be much more elegant and effective, so I would be really interested to see other solutions. :smiley: 

@greatfireball It's unfortunate that you won't be able to join us for the next HackyHour. :disappointed: . However, I'm really looking forward to see your solution.
